src/averageAges.js

```diff
@@ -7,19 +7,34 @@
  * function calculates average age only for men who died in this century
  *
  * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
+ * Divide year of personson's death by 100: Math.ceil(personson.died / 100)
  *
  * @param {object[]} people
  * @param {number} century - optional
  *
  * @return {number}
  */
+
+function averageAge(persons) {
+  return persons
+    .map(person => person.died - person.born)
+    .reduce((firstPerson, secondPerson) => (
+      firstPerson + secondPerson)) / persons.length;
+};
+
 function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
+  // replace `if ()` statement with &&, || or ?:`
   // without nesting
+  const everyMan = people.filter(person => person.sex === 'm');
+  const thisCenturyPeople = everyMan.filter(person => (
+    Math.ceil(person.died / 100) === century));
```

Comment:
for now you create 2 people arrays for each method call
let's call correct filter method depending on century argument, create only one correct array and will work with it